spec/system/searching_spec.rb

```diff
@@ -121,4 +121,9 @@
     visit "/catalog?f[subject_facet][]=Japan%E2%80%94History"
     expect(page).to have_content '1 entry found'
   end
+
```

Comment:
This test is great.  There is also a test for the helper, which would be a great place to add a test for the syntax of the output https://github.com/pulibrary/orangelight/blob/34bf790fd5fb9aab0cdbbd7f514ff0b5d2c39fa4/spec/helpers/locations_spec.rb#L49